frontend/src/components/Details/Details.css

```diff
@@ -0,0 +1,88 @@
+#details {
```

Comment:
# Frontend review

- [ ] Important: I was unable to install the dependencies and therefore unable to run your frontend project:

![image](https://github.com/sajibarafatsiddiqui/final-capstone-project/assets/57421823/209afda8-3525-4a64-b548-77f4ce53a840)

This won't happen on your end, only for new users because you are the developers.

To reproduce this error and correct it, kindly delete your `node_modules` folder and your `package-lock.json` file.  Then, do an `npm install` again in this branch (redux-setup).

Tip:  It seems that the problem is `mantine` and its dependency versions.

_note:  This means I wasn't able to get into the frontend app to test its functionality.  This will be reviewed on the next review._

- [ ] Kindly, remove the old commented code found in:

1. ListItem.js
2. Login.js
3. MyReservations.js
4. Navigation.js
5. helpers.js
6. redux/authentication/index.js
7. redux/cars/index.js
8. redux/rental/index.js
9. util/ProtectedRoute.js

Please remove the commented code from these files.  Also, remove all the commented code from the project that I may have missed in this list. 👍 

- [ ] Kindly, remove all the debugging `console.log` statements from these files.

![image](https://github.com/sajibarafatsiddiqui/final-capstone-project/assets/57421823/a58e16cc-6966-472e-a652-8c6cb09694b5)

That way your repo's cleanliness will be improved. 👍 

- [ ]  Please, kindly refactor this and all other functions in your project to ES6 arrow functions, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry:

![image](https://github.com/sajibarafatsiddiqui/final-capstone-project/assets/57421823/9b054000-ce07-470d-a289-7c0063b98b53).

That way, your code will meet the latest standards. 💪

## Suggestions and extra non-actionable feedback

1. You coerced both projects (the API and the Frontend) on a single repo by using different folder names (backend and frontend).  99% of the other students and in the industry, when we have a project that spans 2 apps, we do a different repo for each app.  Kindly consider this for next time.

2. The review request pointed to the `details` branch which was the only pull request at the time of this review.  I noticed this branch had changes over a week ago:

![image](https://github.com/sajibarafatsiddiqui/final-capstone-project/assets/57421823/3aff66c4-e614-41d9-a628-17fa373c70fa)


It didn't contain a readme file. Upon contacting and asking the team, I discovered the correct and most recent branch was `redux-setup`:

![image](https://github.com/sajibarafatsiddiqui/final-capstone-project/assets/57421823/0f397371-771a-4ee1-9529-fe6f45f08a42)

Therefore, next time, kindly submit the link to the correct branch and ensure that it contains the whole project to review.

These suggestions are for the future so that they happen here in school only, and not in a future job. 👍